libs/ui/src/admin/ArtApprovalModal/ArtApprovalForm.tsx

```diff
@@ -0,0 +1,221 @@
+import { FC, memo, useState } from 'react'
+
+import {
+  Avatar,
+  Button,
+  ButtonGroup,
+  HStack,
+  Text,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalOverlay,
+  Stack,
+  Textarea,
+  Box,
+  VStack,
+  Image,
+  Center,
+  Flex,
+} from '@chakra-ui/react'
+import { Splide, SplideSlide } from '@splidejs/react-splide'
+import {
+  HiDotsVertical,
+  HiOutlineCheck,
+  HiOutlineX,
+  HiPencil,
+} from 'react-icons/hi'
+
+import { ArtImageProps, ArtApprovalFormTypes } from './types'
+
+const ArtImage: FC<ArtImageProps> = memo(({ image, alt }) => {
+  return (
+    <Image
+      objectFit="contain"
+      src={`${process.env['NX_API_URL']}${image.url}`}
+      alt={alt}
+    />
+  )
+})
+
+export const ArtApprovalForm: FC<ArtApprovalFormTypes> = ({
+  onReject,
+  onApprove,
+  onDelete,
+  artId,
+  artDescription,
+  artTitle,
+  artImages,
+  editorId,
+  isOpen,
+  onClose,
+  // editorAvatar,
+  // editorName,
+  artArtistName,
+  isEdit,
+}) => {
+  const [feedback, setFeedback] = useState('')
+
+  const handleReject = (artId: number, editorId: number, feedback: string) => {
+    onReject(artId, editorId, feedback)
+  }
+
+  const handleApprove = (artId: number, editorId: number, feedback: string) => {
+    onApprove(artId, editorId, feedback)
+  }

```

Comment:
```suggestion
  const handleApprove = () => {
    onApprove(artId, editorId, feedback)
  }
```

Click functions doesn't have to have parameters, since we get them from state or props. For example we have `artId, editorId` from props and `feedback` from the state. You don't need to pass them again:

```tsx
// Instead of
onClick={() => handleApprove(...)}
onClick={handleApprove)
```